packages/ui/src/validators/types/Validator.ts

```diff
@@ -0,0 +1,14 @@
+import BN from 'bn.js'
+
+import { Address } from '@/common/types'
+import { MemberWithDetails } from '@/memberships/types'
+
+export interface Validator {
+  member?: MemberWithDetails
+  address: Address
+  isVerified: boolean
+  isActive: boolean
+  totalRewards: BN
+  APR: number
+  startedOn: string
```

Comment:
I confirmed with @dmtrjsg that we can just remove the "started on" column.
```suggestion
```

Reply:
I got it